api/open_api_3.yaml

```diff
@@ -158,6 +158,25 @@ paths:
         '500':
           $ref: '#/components/responses/StatusInternalServerError'  
   /list:
+    parameters: 
```

Comment:
Note: I'm planning to refactor the `page` & `pageSize` params into some sort of shared `Paginate` component that we can reference here instead since I imagine we share these in a lot of places -- was just playing around here to learn the OpenAPI spec